modules/ROOT/pages/endpoint-sftp-send.adoc

```diff
@@ -27,27 +38,27 @@ Send application messages (JSON or XML) to your backend applications in inbound
 |Yes
 
 |Username
-|Username for the SFTP Server.
+|The username for the SFTP Server.
 |Yes
 
 |Password
-|Password for the SFTP Server.
+|The password for the SFTP Server.
```

Comment:
Why capitalize "server" in description? ditto prev.

Reply:
Done.